.github/workflows/pr-e2e-gate.yaml

```
@@ -406,22 +397,22 @@ jobs:
-      name: approve-credentialed-e2e-for-internal-pr
+      name: ${{ needs.coordinate.outputs.approval_environment }}
       deployment: false
     permissions:
       actions: write
       checks: write
       contents: read
       pull-requests: read
     concurrency:
-      group: pr-e2e-gate-${{ github.repository }}-${{ needs.coordinate.outputs.control_plane_approval_pr_number }}-${{ needs.coordinate.outputs.control_plane_approval_head_sha }}-${{ needs.coordinate.outputs.control_plane_approval_base_sha }}
+      group: pr-e2e-gate-${{ github.repository }}-${{ needs.coordinate.outputs.approval_pr_number }}-${{ needs.coordinate.outputs.approval_head_sha }}-${{ needs.coordinate.outputs.approval_base_sha }}
       queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)